### PR TITLE
Close #137: Fix parsing of tf.nn.local_response_normalization function

### LINKED ISSUE #137: Incorrect conversion of tf.nn.local_response_normalization function
I am trying to save and to convert a simple python network:
```
import tensorflow as tf
import nnef_tools.io.tf.graphdef as graphdef


def testnet_local_response_normalization():
    x = tf.placeholder(tf.float32, shape=[3, 4, 5, 6], name='input')
    return tf.nn.local_response_normalization(x, depth_radius=2, bias=1, alpha=1/5, beta=0.5)


tf.reset_default_graph()
with tf.Session() as sess:
    result = testnet_local_response_normalization()
    sess.run(tf.global_variables_initializer())
    graphdef.save_default_graph("model.pb", session=sess, outputs={result: "output"})

```
using commands:
```
python ./test.py 
python -m nnef_tools.convert --input-format=tf --output-format=nnef --input-model=./model.pb --output-model=model.nnef
```
the conversion result is:
```
version 1.0;

graph G(external1) -> (copy1)
{
    external1 = external<scalar>(shape = [3, 4, 5, 6]);
    local_response_normalization1 = local_response_normalization(external1, size = [1, 2, 1, 1], alpha = 0.20000000298023224, beta = 0.5, bias = 1.0);
    copy1 = copy(local_response_normalization1);
}
```
But the [documentation](https://www.tensorflow.org/api_docs/python/tf/nn/local_response_normalization) for the `tf.nn.local_response_normalization` function has some differences from the NNEF [documentation](https://www.khronos.org/registry/NNEF/specs/1.0/nnef-1.0.3.html#normalization-operations) for the `local_response_normalization` operation:
1. In tf normalization is done on channel dimension, that equals 6 in python example
2. In tf normalization size is set by depth_radius, not by full normalization size
3. In tf a squared sum is not divided by normalization size

According to these differences the conversion result should be:
```
version 1.0;

graph G(external1) -> (transpose2)
{
    external1 = external<scalar>(shape = [3, 4, 5, 6]);
    transpose1 = transpose(external1, axes = [0, 3, 1, 2]);
    local_response_normalization1 = local_response_normalization(transpose1, size = [1, 5, 1, 1], alpha = 1.0000000149011612, beta = 0.5, bias = 1.0);
    copy1 = copy(local_response_normalization1);
    transpose2 = transpose(copy1, axes = [0, 2, 3, 1]);
}
```
I have created a pull request that fixes these differences, but I am not sure about correctness of a tensors transposing.

### PR DESCRIPTION
**Pull request by dvorotnev**

### CODE REVIEW

**Review by gyenesvi — CHANGES_REQUESTED** [review]
The size/alpha calculation looks good (I remember this difference but this was overlooked in the latest migration of the converter). However, the output transpose is redundant, the `transpose_output(O[0])` is enough, the `transpose_like` is not needed then.

**Comment by dvorotnev** [comment]
Thanks! I fixed the transposition in the amended commit.
